acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -291,6 +298,25 @@ public void waitForNewFinalization() {
         MINUTES);
   }
 
+  public void waitForNonDefaultExecutionPayload() {
+    LOG.debug("Wait for a block containing a non default execution payload");
+
+    waitFor(
+        () -> {
+          final Optional<SignedBlock> block = fetchHeadBlock();
+          assertThat(block).isPresent();
+          assertThat(block.get()).isInstanceOf(SignedBeaconBlockBellatrix.class);
+
+          final SignedBeaconBlockBellatrix bellatrixBlock = (SignedBeaconBlockBellatrix) block.get();
+          final ExecutionPayload executionPayload = bellatrixBlock.getMessage().getBody().executionPayload
+              .asInternalExecutionPayload(spec, bellatrixBlock.getMessage().slot)
+              .orElseThrow();
+
+          assertThat(executionPayload.isDefault()).isFalse();
+          LOG.debug("Non default execution payload found at slot " + bellatrixBlock.getMessage().slot);
+        }, 300, MINUTES);
```

Comment:
nit: need to set a more realistic time for wait here. The intent is basically a time at which to call it a failure. If a normal message should come within a minute, then i could see a 5 minute timeout being reasonable, but 300 minutes would mean 1 commit in a day can be verified potentially without manually going in and stopping jobs...

Reply:
Nice catch! I got the unit wrong here.. was thinking to do 5 minutes :)  Fixed this now. Thanks @rolfyone !

Reply:
oh that makes sense, usually we just do 5 minutes like below :) 300 seconds would also work :)